hera_qm/tests/test_xrfi.py

```diff
@@ -48,27 +48,30 @@
 def test_uvdata():
     uv = UVData()
     uv.read_miriad(test_d_file)
+    uv.use_future_array_shapes()
     xant = uv.get_ants()[0]
     xrfi.flag_xants(uv, xant)
-    assert np.all(uv.flag_array[uv.ant_1_array == xant, :, :, :])
-    assert np.all(uv.flag_array[uv.ant_2_array == xant, :, :, :])
+    assert np.all(uv.flag_array[uv.ant_1_array == xant, :, :])
+    assert np.all(uv.flag_array[uv.ant_2_array == xant, :, :])
 
 
 def test_uvcal():
     uvc = UVCal()
     uvc.read_calfits(test_c_file)
+    uvc.use_future_array_shapes()
     xant = uvc.ant_array[0]
     xrfi.flag_xants(uvc, xant)
-    assert np.all(uvc.flag_array[0, :, :, :, :])
+    assert np.all(uvc.flag_array[0, :, :, :])
 
 
 def test_uvflag():
     uvf = UVFlag(test_f_file)
+    uvf.use_future_array_shapes()
```

Comment:
There's actually also a way to do this in one line, you can do:
`uvf = UVFlag(test_f_file, use_future_array_shapes=True)`

We're adding a similar option to UVData and UVCal read and from_file methods in an upcoming PR.